Android.mk

```diff
@@ -46,7 +46,14 @@ ifeq ($(TARGET_BUILD_VARIANT),user)
 endif
 
 ifeq ($(TARGET_BUILD_VARIANT),userdebug)
-    KERNELFLINGER_CFLAGS += -DUSERDEBUG
+    KERNELFLINGER_CFLAGS += -DUSERDEBUG -DKERNELFLINGER_LOG_FILE
```

Comment:
Why don't you create a nice` KERNELFLINGER_LOG_OUTPUT `variable which can be `serial` or `file` for instance ?

Reply:
Two macros were removed, and create a KERNELFLINGER_LOG_OUTPUT variable.